include/internal/quic_rx_depack.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_QUIC_RX_DEPACK_H
+# define OSSL_QUIC_RX_DEPACK_H
+
+int ossl_quic_handle_frames(SSL *ssl, OSSL_QRX_PKT *qpacket);
+__owur int ossl_quic_depacketize(SSL *ssl);
```

Comment:
True, we don't.  Our internal QUIC headers are rather...  topical.  That's understandable, considering that we've split it into individual parts and naturally want to avoid merge conflicts :wink: